lisa/sut_orchestrator/hyperv/platform_.py

```diff
@@ -334,3 +378,255 @@ def _delete_node(node_ctx: NodeContext, wait_delete: bool) -> None:
                 for node in environment.nodes.list()
             ]
         )
+
+
+class HyperVDevicePoolImpl(BaseDevicePoolImpl):
+    def __init__(
+        self,
+        node: RemoteNode,
+        assignable_device_script: PurePath,
+        runbook: HypervPlatformSchema,
+    ) -> None:
+        # Device Passthrough configs
+        # Mapping of Host Device Passthrough
+        self.AVAILABLE_HOST_DEVICES: Dict[
+            HostDevicePoolType, List[DeviceAddressSchema]
+        ] = {}
+        self.SUPPORTED_HOST_DEVICE_POOLTYPE = [
```

Comment:
Should this be a static variable instead of a member variable?

Reply:
@cwize1 

Yes, idea is to put platform level check for supported pool type. I will move this out of constructor as static member.